flocker/cli/script.py

```diff
@@ -69,11 +76,13 @@ def parseArgs(self, deployment_config, application_config):
             raise UsageError('No file exists at {path}'
                              .format(path=application_config.path))
 
-        self["deployment_config"] = deployment_config.getContent()
+        self["url"] = b"http://{}:{}/configuration/_compose".format(
+            control_host, self["port"])
```

Comment:
:question: 

`b"...".format(...)` seems so very wrong to me.  On the other hand, it's *not* wrong...  At least, judged by whether it does the right thing...